domains/client/domain-operator/src/domain_worker.rs

```diff
@@ -205,8 +303,29 @@ pub(super) async fn start_worker<
         info!("🧑‍ Running as Full node...");
         drop(new_slot_notification_stream);
         drop(acknowledgement_sender_stream);
-        while let Some(maybe_block_info) = throttled_block_import_notification_stream.next().await {
+
+        'import_loop: while let Some(maybe_block_info) =
+            throttled_block_import_notification_stream.next().await
+        {
             if let Some(block_info) = maybe_block_info {
+                println!("Inside operator processing: {block_info:?}");
+
+                let cache_result = maybe_cache_block_info(
+                    block_info.clone(),
+                    target_block_number,
+                    snap_sync_orchestrator.clone(),
+                    &mut block_queue,
+                    &bundle_processor,
+                    span.clone(),
+                )
+                .await;
```

Comment:
AFAIU, it will cause a deadlock - we need consensus chain import working because we snap sync consensus chain to the segment starting block and need to import some blocks before we reach the consensus chain block related to the last confirmed domain block.

Reply:
>The caching method ignores the block number less than the target block.

Actually, ignore all the block import notifications until the domain sync is finished, the above linked code will properly handle all the missed block notifications.